MIGRATION_GUIDE.md

```
@@ -0,0 +1,468 @@
+### Build Trigger Summary (Updated)
+**Continuous Integration (Direct Push):**
+- Push to `main` → ✅ Build triggered → Uses **PROD** variables → **Release package**
+- Push to `development/dotnet10` → ❌ **No build** → Must create PR to `main`
+- Push to feature branches → ❌ **No build** → Must create PR to `main`
+
+**Pull Request Validation:**
+- PR targeting `main` (from any branch) → ✅ Build triggered → Uses **ACC** variables → **Preview package** (with `-preview` suffix)
+- PR targeting other branches → ❌ No build triggered
+
+**Workflow Summary:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)